docs/hub/models-uploading.md

```diff
@@ -7,10 +7,14 @@ You can link repositories with an individual user, such as [osanseviero/fashion_
 There are several ways to upload models to the Hub, described below.
 
 - In case your model comes from a library that has [built-in support](#upload-from-a-library-with-built-in-support), one can use the existing methods.
-- In case your model is a custom PyTorch model, the recommended way is to leverage the [huggingface_hub](#using-the-huggingface_hub-client-library) Python library as it allows to add `from_pretrained`, `push_to_hub` and [automated download metrics](https://huggingface.co/docs/hub/models-download-stats) capabilities to your models, just like models in the Transformers, Diffusers and Timm libraries.
+- In case your model is a custom PyTorch model, the two recommended ways to integrate with the ecosystem is to either:
+    - use the `transformers`'s [custom code](https://huggingface.co/docs/transformers/custom_models) feature, which makes your model directly usable in the Transformers library
```

Comment:
this is a bit too transformers specific at this stage no?

Reply:
the rest LGTM from a quick read

Reply:
Agree, and I see this more of a `model comes from a library that has ...` class. E.g. we also have remote code with `diffusers`.